docs/user-guide/share.md

```diff
@@ -35,13 +35,18 @@ The **Social** section allows the user to share the resource on the most common
 
 The **Embed** section provides to the user the needed snippets, **embedded code** or the **MS APIs** (only available for *maps*) to embed MapStore in a third party web page.
 
-<img src="../img/share/embed.jpg" class="ms-docimage"  style="max-width:400px;"/>
+<img src="../img/share/embed.jpg" class="ms-docimage"  style="max-width:400px;"/> 
 
-In addition, [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) provides two options to customize a bit the *embedded code*:
+In addition, [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) provides options to customize a bit the *embedded code*:
+
+* The user can configure **height** and **width** of the embedded resource by choosing `Small` (*600x500*), `Medium` (*800x600*), `Large` (*1000x800*) and `Custom`
```

Comment:
* The user can configure **height** and **width** of the embedded resource by choosing `Small` (*600x500*), `Medium` (*800x600*), `Large` (*1000x800*) and `Custom` (it is possible to choose the desired size).